reference/7/Microsoft.PowerShell.Core/ForEach-Object.md

```diff
@@ -18,150 +18,179 @@ Performs an operation against each item in a collection of input objects.
-  You can use a script block to specify the operation.
-  Within the script block, use the `$_` variable to represent the current object.
-  The script block is the value of the *Process* parameter.
-  The script block can contain any PowerShell script.
+- **Script block**. You can use a script block to specify the operation. Within the script block,
+  use the `$_` variable to represent the current object. The script block is the value of the
+  *Process* parameter. The script block can contain any PowerShell script.
 
-  For example, the following command gets the value of the **ProcessName** property of each process on the computer.
+  For example, the following command gets the value of the **ProcessName** property of each process
+  on the computer.
 
   `Get-Process | ForEach-Object {$_.ProcessName}`
 
-- **Operation statement**.
-  You can also write an operation statement, which is much more like natural language.
-  You can use the operation statement to specify a property value or call a method.
-  Operation statements were introduced in Windows PowerShell 3.0.
+- **Operation statement**. You can also write an operation statement, which is much more like
+  natural language. You can use the operation statement to specify a property value or call a
+  method. Operation statements were introduced in Windows PowerShell 3.0.
 
-  For example, the following command also gets the value of the **ProcessName** property of each process on the computer.
+  For example, the following command also gets the value of the **ProcessName** property of each
+  process on the computer.
 
   `Get-Process | ForEach-Object ProcessName`
 
-  When using the script block format, in addition to using the script block that describes the operations that are performed on each input object, you can provide two additional script blocks.
-  The Begin script block, which is the value of the *Begin* parameter, runs before this cmdlet processes the first input object.
-  The End script block, which is the value of the *End* parameter, runs after this cmdlet processes the last input object.
+  When using the script block format, in addition to using the script block that describes the
+  operations that are performed on each input object, you can provide two additional script blocks.
+  The Begin script block, which is the value of the **Begin** parameter, runs before this cmdlet
+  processes the first input object. The End script block, which is the value of the **End**
+  parameter, runs after this cmdlet processes the last input object.
+
+- **Parallel running script block**. Beginning with PowerShell 7.0, a third parameter set is
+  available that runs each script block in parallel. There is a `-ThrottleLimit` parameter that
+  limits the number of parallel scripts running at a time. As before, use the `$_` variable to
+  represent the current input object in the script block. Use the `$using:` keyword to pass variable
+  references to the running script.
```

Comment:
Below a note is about efficiency but here is more right place to say that it requires a lot of overhead. Perhaps a short quote from your blog post would be enough.